File: Level1/제일 작은 수 제거.py
# 제일 작은 수 제거
def solution(arr):
    answer = []
    min_value=min(arr) #리스트 안에 제일 작은 값
    arr.remove(min_value)
    if len(arr)!=0: #리스트 원소가 하나인 것을 대비
        for i in arr:
            answer.append(i)
    else:
        answer.append(-1)
    return answer
arr=list(map(int,input().split()))
print(solution(arr))

File: Level1/제일 작은 수 제거.cpp
#include <string>
#include <vector>

using namespace std;

int find_minidx(vector<int> v) {
    int min_idx = 0;
    for (int i = 1; i < v.size(); i++) {
        //더 작은 값을 발견하면
        if (v[i] < v[min_idx])
            min_idx = i;
    }
    return min_idx;
}

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    if (arr.size() == 1) {
        answer.push_back(-1);
        return answer;
    }
    int min_idx = find_minidx(arr);
    for (int i = 0; i < arr.size(); i++) {
        if (min_idx == i)
            continue;
        answer.push_back(arr[i]);
    }
    return answer;
}